### PR TITLE
- Updated to include jasmine-reports for jasmine 2.0

### PR DESCRIPTION
**Pull request by johnsonw**
- Updated version to 0.0.4
- Updated index file to allow the generation of junit reports.
- Added editor config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "istanbul": "0.2.12",
     "minijasminenode2": "1.0.0",
     "multi-glob": "0.4.0",
-    "yargs": "1.2.5"
+    "yargs": "1.2.5",
+    "jasmine-reporters": "git://github.com/larrymyers/jasmine-reporters.git#jasmine2"
```
Comment (wayneseymour) [inline package.json:26]: May need to be an actual npm repo, not a git repo.

File: index.js [hunk 2]
```diff
@@ -31,6 +31,15 @@ if (specs)
 else
   execute();
 
-function execute () {
+function execute() {
+  if (argv.reportType === 'junit') {
+    var jasmineReporters = require('jasmine-reporters');
```
Comment (jgrund) [inline index.js:36]: It won't hurt to require this early. Probably preferable as require is synchronous.

Comment (johnsonw) [inline index.js:36]: Ok, I will require this up at the top.

File: package.json (hunk 1 above)
Comment (johnsonw) [inline package.json:26]: I left a message on the jasmine-reporters github page to see when they are going to merge their changes into master. Unfortunately I think I'm stuck with this at the moment unless we use your chum bucket. Why don't we try it like this and if it doesn't work we setup chum bucket.

File: index.js (hunk 2 above)
Comment (wayneseymour) [inline index.js:36]: Hmm, I seem to recall Julie mentioning you couldn't require this early.

Comment (jgrund) [inline index.js:36]: Let's give it a shot and see what happens.

**Pull request closed without merging**